### PR TITLE
Issue #34: datalinks / correlation

### PR DESCRIPTION
**Pull request by idrissneumann**

### CODE REVIEW

**Comment by idrissneumann** [comment]
It works !

Screenshot 2023-12-08 at 15 26 33

Screenshot 2023-12-08 at 15 24 59

**Comment by idrissneumann** [comment]
Fixed, there's no double links anymore

Screenshot 2023-12-11 at 11 28 04

File: src/configuration/DataLink.tsx [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+                datasourceUid: ds.uid,
+              });
+            }}
+            current={value.datasourceUid}
+          />
+        )}
+      </div>
+
+      <div className={styles.row}>
+        <InlineField label="Base64" labelWidth={12} title="Base64 traceId">
```
Comment (fmassot) [inline src/configuration/DataLink.tsx:142]: Can you put the switch just below the "field" input?
Also "Base64" is not very understandable, we could say "Field encoded in base64"
and add a tooltip to explain that quickwit encodes the traceID in base64 by default and Jaeger uses hex

Comment (idrissneumann) [reply src/configuration/DataLink.tsx:142]: Yes I thought about putting a more longer text but we'll have to expend the size of the label with more columns. I'll pick you a new screenshot so you'll able to tell me if it's better or not.

Comment (idrissneumann) [reply src/configuration/DataLink.tsx]: @fmassot Is-it better like that ?

Screenshot 2023-12-13 at 11 53 34

Screenshot 2023-12-13 at 11 53 41

Comment (fmassot) [reply src/configuration/DataLink.tsx]: yes!

**Pull request merged**